x-pack/plugin/esql/qa/testFixtures/src/main/resources/where-like.csv-spec

```diff
@@ -319,3 +319,63 @@ warningRegex:java.lang.IllegalArgumentException: single-value function encounter
 emp_no:integer | job_positions:keyword 
 10025          | Accountant 
 ;
+
+likeWithUpper
+FROM employees
+| KEEP emp_no, first_name
+| SORT emp_no
+| WHERE TO_UPPER(first_name) LIKE "GEOR*"
```

Comment:
Could you please add a couple of negative tests, eg. where TO_UPPER tries to match a lowercase pattern
```
FROM employees
| KEEP emp_no, first_name
| SORT emp_no
| WHERE TO_UPPER(first_name) LIKE "geor*"
```

Reply:
Added more tests (the logical folding is tested as well)